Expensify/App
# Inset the full screen modal used for workspaces on large devices

cc @marcaaron 

### Details
Added the expected spacing around the full-screen modals without modifying too many things. 

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/169135

### Tests / QA Steps

Using a large screen (like the web version or the desktop app):
1. Create a new workspace
2. Go to its settings page
3. Verify that the full-screen modal is separated from the window border and that its corners are rounded, like this photo:
![image](https://user-images.githubusercontent.com/6759612/124625576-3583ae80-de7e-11eb-81f5-eaeb42f380db.png)

Using a small screen (mobile version):
1. Create a new workspace
2. Go to its settings page
3. Verify that the full-screen modal is not separated from the window border and has no rounded corners, like in this photo:
![image](https://user-images.githubusercontent.com/6759612/124625075-c60dbf00-de7d-11eb-885a-9d4e07c03e36.png)

### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/6759612/124625576-3583ae80-de7e-11eb-81f5-eaeb42f380db.png)

#### Mobile Web
![image](https://user-images.githubusercontent.com/6759612/124625647-4502f780-de7e-11eb-97dc-a4d334dfd62f.png)

#### Desktop
![image](https://user-images.githubusercontent.com/6759612/124625916-83001b80-de7e-11eb-9110-f78e1aa518c6.png)

#### iOS
![image](https://user-images.githubusercontent.com/6759612/124625075-c60dbf00-de7d-11eb-885a-9d4e07c03e36.png)

#### Android
![image](https://user-images.githubusercontent.com/6759612/124623314-2e5ba100-de7c-11eb-90d2-30ddbf63a01d.png)
